src/radio/sx126x/radio.c

```diff
@@ -1072,6 +1083,7 @@ void RadioRx( uint32_t timeout )
     {
         TimerSetValue( &RxTimeoutTimer, timeout );
         TimerStart( &RxTimeoutTimer );
+        RxKernelTimer = k_uptime_get();

```

Comment:
Timer should not be counted when timeout == 0?

Reply:
0 is continuous mode and not sure if we are ever in continuous mode.